src/main/java/com/werfen/report/test/excel/ExcelComparator.java

```diff
@@ -0,0 +1,102 @@
+package com.werfen.report.test.excel;
```

Comment:
No related to here, but I miss adding some continuous integration to guarantee in the PRs that this works as expected. I would suggest using the same approach as in the java-generator-library (using GitHub Actions)

Reply:
I'm not following you in that point. Can we discuss about it?